CONTRIBUTING.adoc

```diff
@@ -0,0 +1,93 @@
+= Contributing to Plugin Health Scoring
+
+This page provides information about contributing to Plugin Health Scoring code base.
+
+== Getting started
+
+The project is using Maven with Java 17 and Yarn to be built.
+The entire build process can be handled by Maven.
+Yarn build will bundle the front-end files only.
+
+. Clone the repository
```

Comment:
```
. Clone the repository
```
. Clone the repository.

adding a full stop since some of these items have a "."